src/stash-list/folder.vue

```diff
@@ -127,6 +173,11 @@ export default defineComponent({
             return getDefaultFolderNameISODate(this.folder.title) !== null
                 ? undefined : this.folder.title;
         },
+        tooltip(): string {
+            const st = this.childTabStats;
+            const statstip = `(stashed: ${this.validChildren.length}, tabs: ${st.open}/${st.discarded}/${st.hidden} open/unloaded/hidden)`;
+            return `${this.nonDefaultTitle ?? this.defaultTitle} ${statstip}`;
```

Comment:
Suggest separating these with `\n` instead of a space, so that it's more visually obvious where the title ends and the stats begin.